fonts.css

```diff
@@ -65,3 +65,10 @@
   font-size: 20px;
   line-height: 24px;
 }
+.collection-title__adaptiv {
+  font-family: Arial, sans-serif, Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5608084)